apis/replication.storage/v1alpha1/volumegroupreplication_types.go

```diff
@@ -72,6 +73,10 @@ type VolumeGroupReplicationSource struct {
 // VolumeGroupReplicationStatus defines the observed state of VolumeGroupReplication
 type VolumeGroupReplicationStatus struct {
 	VolumeReplicationStatus `json:",inline"`
+	// PersistentVolumeClaimsRefList is the list of PVCs for the volume group replication.
+	// The maximum number of allowed PVCs in the group is 100.
```

Comment:
@Madhu-1, What is the rationale for setting the maximum number of PVCs in a group to 100?

Reply:
This is the support max number of PVC in a group. we already have this in VGRC for PV object